backend/routes/api/users.js

```diff
@@ -0,0 +1,194 @@
+const express = require("express");
+const router = express.Router();
+const User = require("../../models/User");
+const { celebrate, Joi } = require("celebrate");
+
+// get all users
+router.get(
+  "/",
+  celebrate({
```

Comment:
What happens if the data passed doesn't adhere to the celebrate validation schema?

Reply:
It's a very cool middleware—it'll return a Boom error object like [this](https://hapi.dev/family/boom/?v=8.0.1#boombadrequestmessage-data) automatically. Meaning that we can assume that when any of the contents of the function is run, we know the data was correctly validated.